src/graphql/resolveConnection.js

```diff
@@ -8,6 +8,7 @@ const resolveConnection = (
   getFromClause = constant(table.getIdentifier()),
 ) => {
   const columns = table.getColumns()
+  const fullPrimaryKey = table.getPrimaryKeys()
```

Comment:
I changed it anyway, I just didn’t want it to get accidently used so I named it something more opaque.